api/application.go

```diff
@@ -194,7 +194,7 @@ func (h ApplicationHandler) List(ctx *gin.Context) {
 		_ = ctx.Error(err)
 		return
 	}
-	filter = filter.Renamed("platform.id", "PlatformId")
+	filter = filter.Renamed("platform.id", "platform_id")
 
```

Comment:
Curious about the field names change to snake-case, is this something want to do to have better readable/focusing on some conventions, or support 2 databases are forcing us to do that?